src/Rpc/ErrorInfo.php

```diff
@@ -98,43 +100,44 @@ public function __construct($data = NULL) {
     }
 
     /**
-     * The type of the error. This is a constant value that identifies the
-     * proximate cause of the error. Error types are unique within a particular
-     * source of errors. This should be at most 63 characters and match
+     * The reason of the error. This is a constant value that identifies the
+     * proximate cause of the error. Error reasons are unique within a particular
+     * domain of errors. This should be at most 63 characters and match
      * /[A-Z0-9_]+/.
      *
-     * Generated from protobuf field <code>string type = 1;</code>
+     * Generated from protobuf field <code>string reason = 1;</code>
      * @return string
      */
-    public function getType()
+    public function getReason()
```

Comment:
This appears to be a BC break. I am not sure why they would change `type` to `reason` like this, but it's sure to break things if we release this PR without a major version bump.

I'd be really curious what caused this change and how it should be handled, as this is not typically something our protos do.

Reply:
It looks like the change was made back in March 2020 in order to maintain consistency with the Google REST APIs: https://b.corp.google.com/issues/151169088

I'm not sure how it should be handled either :sweat_smile: I can try asking other language repos what they did with this change.

Reply:
So it looks like Java, python, and ruby didn't have this message until after the breaking change.

Go's proto binding monorepo is unversioned so it just "took" the breaking change (kind of a live on head philosophy for this, as weird as it is).